.github/workflows/ci.yml

```diff
@@ -5,16 +5,26 @@ on:
     branches:
       - main
   push:
-    branches:
-      - main
+    branches: [main, development]
```

Comment:
@tdonaworth, I'm late for the review, but is there a reason that we are in including pushes to `development`?

The `dev_deploy.yml` already run the backend/frontend unit tests and end-to-end tests before allowing the deploy.

Reply:
unit-tests and e2e tests are run, but no additional security checks are run; This specifically occurs for dependency related security findings that may pop up in the time between when `main` was pushed, and when `deploy` is performed; there could be a few days, when the security posture has changed, etc. 

Under this same line of thinking; I plan on creating a nightly job which runs those type of security checks.

Reply:
Ah, OK.  Should we then actually put these two security checks as part of a requirement before deployment then?  E.g. add it in the `dev_deploy.yml` file.  That way it will actually stop the deployment if something is found?  Right now if I push to `development`, the deploy will keep happening and if a security finding is found, it isn't reallly reported anywhere except perhaps somehwere in the Security tab.

Albeit, there's a discussion to be had that this is the development environment and used for quick and dirty testing, and therefore one may not care to have development blocked by a finding.

Reply:
Much larger discussion; but yes it should happen and and be required there. Won't lie I haven't looked too closely at our `dev_deploy` yet; especially from a `path to production` standpoint; so it will change.